applications/RomApplication/tests/fluid_dynamics_test_files/LSPGROM/ProjectParametersLSPGROM.json

```diff
@@ -0,0 +1,98 @@
+{
+    "analysis_stage": "KratosMultiphysics.FluidDynamicsApplication.fluid_dynamics_analysis",
+    "problem_data"     : {
+        "problem_name"  : "couette_flow_test",
+        "parallel_type" : "OpenMP",
+        "echo_level"    : 0,
+        "start_time"    : 0.0,
+        "end_time"      : 100.0
+    },
+    "output_processes" : {
+        "gid_output" : [],
+        "vtk_output" : []
+    },
+    "solver_settings"  : {
+        "model_part_name"             : "FluidModelPart",
+        "domain_size"                 : 2,
+        "solver_type"                 : "Monolithic",
+        "model_import_settings"       : {
+            "input_type"     : "mdpa",
+            "input_filename" : "../../../../FluidDynamicsApplication/tests/CouetteFlowTest/couette_flow_test"
+        },
+        "material_import_settings"    : {
+            "materials_filename" : "../../../../FluidDynamicsApplication/tests/CouetteFlowTest/FluidMaterials.json"
```

Comment:
Though I like reusing the test files from the `FluidDynamicsApplication` as much as possible, I'm afraid of breaking this one in the eventual case that we change or refactor it in the future. Just pointing it out to be aware of it.